dtfs-central-api/src/helpers/amendments/tfm-facility-latest-completed-amendment-to-facility-value.test.ts

```diff
@@ -0,0 +1,129 @@
+import { AMENDMENT_STATUS, TfmFacility, TfmFacilityAmendment } from '@ukef/dtfs2-common';
+import { getLatestCompletedAmendmentToFacilityValue } from './tfm-facility-latest-completed-amendment-to-facility-value';
+import { aTfmFacility, aTfmFacilityAmendment } from '../../../test-helpers';
+
+describe('tfm-facility-latest-completed-amendment-to-facility-value', () => {
+  describe('getLatestCompletedAmendmentFacilityValue', () => {
+    it('returns null when the supplied tfm facility amendments are undefined', () => {
```

Comment:
can these all start with should return please

Reply:
File was only temporarily in the diff between the base getting merged into main and main getting merged into this.